spec/acceptance/basic_spec.rb

```diff
@@ -117,3 +117,44 @@ class { 'pulpcore':
   end
 
 end
+
+describe 'with content cache enabled' do
+  certdir = '/etc/pulpcore-certs'
+
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      class { 'pulpcore':
+        cache_enabled => true,
+      }
+      PUPPET
+    end
+  end
+
+  describe service('httpd') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe service('pulpcore-content') do
+    it { is_expected.to be_enabled }
+    it { is_expected.to be_running }
+  end
+
+  describe port(80) do
+    it { is_expected.to be_listening }
+  end
+
+  describe port(443) do
+    it { is_expected.to be_listening }
+  end
+
+  describe port(6379) do
+    it { is_expected.to be_listening }
+  end
```

Comment:
In addition to the port, I'd also like to check the `service('redis')` (or is it `rh-redis5-redis` on EL7?).

And on a related note: I think we should think about using a unix socket for Redis.

Reply:
Done.